test/units/modules/remote_management/lxca/test_lxca_nodes.py

```diff
@@ -0,0 +1,95 @@
+import json
+
+import pytest
+from units.compat import mock
+from ansible.module_utils import basic
+from ansible.module_utils._text import to_bytes
+from ansible.modules.remote_management.lxca import lxca_nodes
+from ansible.modules.remote_management.lxca.lxca_nodes import setup_conn
+from ansible.modules.remote_management.lxca.lxca_nodes import close_conn
+
+
+@pytest.fixture(scope='module')
+@mock.patch("ansible.modules.remote_management.lxca.lxca_nodes.close_conn", autospec=True)
+def setup_module(close_conn):
+    close_conn.return_value = True
+
+
+class TestMyModule():
+    @pytest.mark.parametrize('patch_ansible_module',
+                             [
+                                 {},
+                                 {
+                                     "auth_url": "https://10.240.14.195",
+                                     "login_user": "USERID",
+                                 },
+                                 {
+                                     "auth_url": "https://10.240.14.195",
+                                     "login_password": "Password",
+                                 },
+                                 {
+                                     "login_user": "USERID",
+                                     "login_password": "Password",
+                                 },
+                             ],
+                             indirect=['patch_ansible_module'])
+    @pytest.mark.usefixtures('patch_ansible_module')
+    @mock.patch("ansible.modules.remote_management.lxca.lxca_nodes.setup_conn", autospec=True)
+    @mock.patch("ansible.modules.remote_management.lxca.lxca_nodes.execute_module", autospec=True)
+    def test_without_required_parameters(self, _setup_conn, _execute_module, mocker, setup_module):
+        """Failure must occurs when all parameters are missing"""
+        with pytest.raises(SystemExit):
```

Comment:
This doesn't check that a failure occurred because both `module.fail_json` and `module.exit_json` mpdules raise `SystemExit` exception. In order to check that, either use [`capfd`](https://github.com/ansible/ansible/blob/c53e3ca8e9834259b79c6b8dec9cf8219c4ae113/test/units/modules/packaging/os/test_rhn_register.py#L91) fixture or mock [`AnsibleModule`](https://github.com/ansible/ansible/blob/c53e3ca8e9834259b79c6b8dec9cf8219c4ae113/test/units/modules/source_control/test_gitlab_hooks.py#L72).